### PR TITLE
Fixes issue #477

### PR DESCRIPTION
**Pull request by asharonbaltazar**
### Summary 
Addresses issue #477 

Changed `a` tags to `div`s and removed the `href`, supplementing with `useHistory` hook from `react-router`. Changed respective SCSS. Added new click handler function for brevity in the actual JSX. 

### Test Plan 
Open to ideas.

### Breaking Changes 

None

- [ ] I have updated the documentation accordingly.
- [ ] My PR adds a @ts-ignore

### CODE REVIEW

**Review by ngwattcos — CHANGES_REQUESTED** [review]
Looks good, @asharonbaltazar ! Thanks for taking on this PR.

There is one tiny styling nit: please see the comment

File: src/components/includes/CalendarHeader.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
+import {useHistory} from 'react-router'
```
Comment (ngwattcos) [inline src/components/includes/CalendarHeader.tsx:2]: Please add a space between the brackets for `useHistory`.

**Review by ngwattcos — CHANGES_REQUESTED** [review]

File: src/components/includes/CalendarHeader.tsx (hunk 1 above)
Comment (asharonbaltazar) [reply src/components/includes/CalendarHeader.tsx:2]: Oh my, I thought that should've been automatic. No problem. Coming right up.

**Review by ngwattcos — APPROVED** [review]
LGTM

**Pull request closed without merging**